src/main/java/org/broadinstitute/hellbender/tools/walkers/annotator/allelespecific/AS_ReadPosRankSumTest.java

```diff
@@ -37,9 +38,10 @@
  * </ul>
  *
  */
-//FIXME - this whole class is essentially code duplicated from ReadPosRankSumTest
 public class AS_ReadPosRankSumTest extends AS_RankSumTest implements AS_StandardAnnotation {
 
+    final ReadPosRankSumTest readPosRankSumTest = new ReadPosRankSumTest();
```

Comment:
I feel like the *real* way to do this is via a decorator that makes some annotation allele-specific, but I have no idea how to do that in Java.